Gemfile.lock

```diff
@@ -514,7 +511,6 @@ DEPENDENCIES
   mechanize
   minitest
   mysql2 (= 0.3.20)
-  newrelic-redis
```

Comment:
It should also be removed from the Gemfile:

https://github.com/otwcode/otwarchive/blob/f82a1c7f7b28873d93c518849cb15cdfc8649c4d/Gemfile#L130

Reply:
Thanks, forgot to commit it.